### PR TITLE
Final PR to merge dev into master

### PR DESCRIPTION
**Pull request by MattGomb**
In this project and PR, we created the our second capstone project by:

- Choosing the pokémon API and working with the first 151 items from the API (out of the 1154), where each item is a pokémon.
- Creating a main layout to display all the pokémons on cards
- Creating 'comment' and 'reserve' functions and popup displays for them
- Creating a button for liking each pokécard, and a counter that keeps track of all past likes
- Sending and getting information from the API (separate for pokémons and comments/reservations) to display all comments, all reservations, and all likes.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <section id="navbar-container">
+      <nav id="navbar">
+          <div id="logo"></div>
+          <ul id="navbar-links">
+            <li>Pokémons</li>
+            <li>Info</li>
+            <li>Contact</li>
+          </ul>
+      </nav>
+    </section>
```
Comment (mirouhml) [inline src/index.html]: - [x] Please know that this should be a `header` and not a `section`, and the `page-container` should be a `main` and not a `section` as well, the structure should be `header` then `main` then `footer`, kindly fix this. 😉
- [x] Please know that as per the requirements, the website should contain a Home page header and navigation similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md), currently the navigation isn't available it's only text and not a navigation menu, also it should have the pokémon counter on the navigation menu, kindly fix that as well. 😁

  This is how your navigation looks:

  image

  And this is the mockup's navigation:

  image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @MattGomb @Oklukeok @carreraprogrammer, 

Good job so far!

There are some issues that you still need to work on to go to finish the capstone project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (MattGomb) [reply src/index.html]: Thank you for the keen observations! Changes applied!

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle:  🎉 🎉 🎉
Hi @MattGomb @Oklukeok  @carreraprogrammer , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [x] Nice design :white_check_mark: 
- [ ] Good commit messages :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 



## Optional suggestions
- [ Optional ] In group projects It's always good practice for all team members to have equal contributions in terms of commits, a keen look at Github statistics on this project's contributions shows unequal contribution to the project. Please rectify this in future group projects.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**